spec/features/access_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe 'resources access', type: :feature do
+  context 'when the user is not logged in' do
+    it 'can access the login page' do
+      visit login_path
+
+      expect(page.current_path).to eq(login_path)
+    end
+
+    context 'when the user tries to access a restricted page' do
+      before { visit admin_users_path }
+
+      it 'is redirected to the login page' do
+        expect(page.current_path).to eq(login_path)
+      end
+
+      it 'shows an error message' do
+        expect(find('div.alert').text).to eq('Veuillez-vous connecter pour accéder à cette page.')
```

Comment:
```suggestion
        expect(page).to have_content('Veuillez-vous connecter pour accéder à cette page.')
```

en vrai moi je me ferai pas chier et je ferai ça:

```ruby
        expect(page).to have_css('.alert')
```

demain si on change le texte ça casse.

Reply:
Ca me va.